keys-controller.js

```diff
@@ -60,9 +60,16 @@ function factory($scope, $routeParams, brAlertService, brKeyService,
   };
 
   self.init = function(identity) {
+    self.operations = {
+      add: false,
+      remove: false
+    };
     var sessionPromise = brSessionService.get().then(function(session) {
```

Comment:
The use of session here is questionable. `bedrock-angular-session` is not even a listed dependency in bower.json. I realize this isn't part of the patch, but it's dubious use here is likely one reason that this patch turned out the way it is. We'll want to mark this with a TODO for removal.